scripts/driver.requirements.txt

```diff
@@ -14,6 +14,6 @@ markupsafe < 2.1
 Jinja2 >= 2.10.1; python_version <  '3.10'
 Jinja2 >= 2.10.3; python_version >= '3.10'
 # Jinja2 >=2.10, <3.0 needs a separate package for type annotations
-types-Jinja2
+types-Jinja2 >= 2.11.9
 jsonschema >= 3.2.0
-types-jsonschema
+types-jsonschema >= 3.2.0
```

Comment:
I'm sorry, I don't get it. I thought the breakage was from pip picking up a version of types-jsonschema that's too recent for our version of Python. How does a `>=` dependency fix that?

To be clear: I'm not saying this is wrong, just really that I'm not getting it without some additional explanation.

Reply:
`min_requirements.py` transforms all `>=` requirements into `==` requirements' to make sure we don't depend on features introduced in later versions.

Reply:
Ah, thanks for the reminder! Indeed this makes sense now :)